src/test/java/co/cask/gcp/gcs/GCSConfigHelperTest.java

```diff
@@ -15,9 +15,12 @@
  */
```

Comment:
Is it feasible/worth it to also add integration tests for this?

Reply:
yeah, i'll keep the jira open until we add some